homeassistant/components/ridwell/__init__.py

```diff
@@ -82,3 +93,43 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         hass.data[DOMAIN].pop(entry.entry_id)
 
     return unload_ok
+
+
+async def async_migrate_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Migrate an old config entry."""
+    version = entry.version
+
+    LOGGER.debug("Migrating from version %s", version)
+
+    # 1 -> 2: Update unique ID of existing, single sensor entity to be consistent with
+    # common format for platforms going forward:
+    if version == 1:
+        version = entry.version = 2
+
+        ent_reg = er.async_get(hass)
+        [entity_entry] = [
+            e for e in ent_reg.entities.values() if e.config_entry_id == entry.entry_id
+        ]
+        new_unique_id = f"{entity_entry.unique_id}_{SENSOR_TYPE_NEXT_PICKUP}"
+        ent_reg.async_update_entity(entity_entry.entity_id, new_unique_id=new_unique_id)
```

Comment:
We can use this helper:
https://github.com/home-assistant/core/blob/7fe895e554a7411373758291d5a374227a924e01/homeassistant/helpers/entity_registry.py#L878